src/reference/asciidoc/amqp.adoc

```diff
@@ -2045,6 +2045,82 @@ Interested in any stock quotes
 
 The client is declaring another queue via the declareQueue() method on the AmqpAdmin, and it binds that queue to the market data exchange with a routing pattern that is externalized in a properties file.
 
+[[collection-declaration]]
+===== Declaring Collections of Exchanges, Queues, Bindings
+
+Starting with _version 1.5_, it is now possible to declare multiple entities with one `@Bean`, by returing a
+collection.
+
+Only collections where the first element is a `Declarable` are considered, and only `Declarable` elements from such
+collections are processed.
+
+[source, java]
+-----
+@Configuration
+public static class Config {
+
+	@Bean
```

Comment:
In addition: we talked with you recently that all indents in docs must be whitespaces, not tabs. 
See this strange view form GH.